src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/layout.html.twig

```diff
@@ -16,6 +16,10 @@
                                 <a href="{{ profile.url }}">{{ profile.url }}</a>
                             {% else %}
                                 {{ profile.url }}
+                                {% set referer = profile.collector('request').requestheaders.get('referer') %}
```

Comment:
this code does not handle the case where the `request` collector is not registered, while this case was handled previously (see the handling of status code)